packages/webpack/src/plugin.js

```diff
@@ -634,6 +645,17 @@ LavaMoatPlugin.exclude = EXCLUDE_LOADER
 
 module.exports = LavaMoatPlugin
 
+/**
+ * @typedef {Object} ScuttlerObjectConfig
```

Comment:
not even a suggestion, but "just saying": `Object` is the default extension type and can always be omitted from `@typedef`s having `@property` tags.

if the `@typedef` does _not_ extend `Object`, any `@property` tags _will be ignored_.

![image](https://github.com/user-attachments/assets/c78af056-44d9-45aa-b2b1-dd555ec41f9d)